index/server/Dockerfile

```diff
@@ -24,24 +25,8 @@ RUN set -x ; \
 # Modify the permissions on the necessary files to allow the container to properly run as a non-root UID
 RUN mkdir -p /www/data && chmod -R g+rwx /www/data
 
-WORKDIR /app
-
-# Copy over the registry viewer's resources
-COPY --from=registry-viewer --chown=www-data:root /app/next.config.js ./
-COPY --from=registry-viewer --chown=www-data:root /app/public ./public
-COPY --from=registry-viewer --chown=www-data:root /app/.next ./.next
-COPY --from=registry-viewer /app/node_modules ./node_modules
-COPY --from=registry-viewer /app/package.json ./package.json
-COPY --from=registry-viewer --chown=www-data:root /app/config ./config
-
-# Make sure .next is writable
-RUN chmod -R g+rwx /app/.next
-
-# Run the registry viewer in production
-ENV NODE_ENV production
-
-ENV DEVFILE_VIEWER_ROOT /viewer
-ENV DEVFILE_COMMUNITY_HOST false
+# Define headless env var, if set block registry viewer route on index server
+ENV REGISTRY_HEADLESS ${headless}
```

Comment:
We shouldn't hardcode this in the dockerfile, it should be passed in to the container during deploy time